tests/integration/tasks/test_luma.py

```diff
@@ -0,0 +1,10 @@
+from viadot.tasks import LumaIngest
+
+
+def test_luma_ingest():
+    task = LumaIngest(
+        credentials_secret="luma-dev",
+        metadata_dir_path="tests/resources/metadata/new",
```

Comment:
@Rafalz13 I now think we should probably version these "static" metadata files so it's clear with which version of nesso CLI they were generated?